src/ia-menu-slider.js

```diff
@@ -59,13 +65,50 @@ export class IAMenuSlider extends LitElement {
     return this.selectedMenu ? 'open' : 'closed';
   }
 
+  /**
+   * closes menu drawer
+   */
+  closeMenu() {
+    this.open = false;
+    const drawerClosed = new CustomEvent('ItemNavMenuClosed', {
+      detail: this.selectedMenuDetails,
+    });
+    this.dispatchEvent(drawerClosed);
+  }
+
+  /**
+   * renders menu header
+   */
+  get renderMenuHeader() {
+    // const { selectedMenu } = this;
+    const { title = '', menuDetails = '', actionButton } = this.selectedMenuDetails || {};
+    const actionSection = actionButton
+      ? html`<div class="custom-action">${actionButton}</div>`
+      : nothing;
+
+    return html`
+      <header>
+        <div class='details'>
```

Comment:
SUGGESTION: Purely cosmetic, but in keeping consistent with your convention below, maybe want to use double-quotes for this?

Reply:
thanks for catching.